src/apprt/gtk/Window.zig

```diff
@@ -355,6 +360,13 @@ pub fn updateConfig(
     self: *Window,
     config: *const configpkg.Config,
 ) !void {
+    // avoid multiple reconfigs when we have many surfaces contained in this
+    // surface using the integer value of config as a simple marker to know if
```

Comment:
s/surface/window? or is that what you meant

Reply:
DOH!